ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/core/StandardComponentInitializer.java

```diff
@@ -497,7 +498,11 @@ public void initialize() {
         loaderToolbarGroup.add(loaderAction);
 
         // Define hot-keys
-        keyBinding.getGlobal().addKey(new KeyBuilder().action().alt().charCode('n').build(), "navigateToFile");
+        if(UserAgent.isMac()){
+            keyBinding.getGlobal().addKey(new KeyBuilder().action().alt().charCode('N').build(), "navigateToFile");
```

Comment:
But If 'N' will be work for Linux and Windows and Mac why do we need redundant checking for MacOS